x-pack/plugins/uptime/public/components/monitor/status_details/availability_reporting/__snapshots__/location_status_tags.test.tsx.snap

```diff
@@ -996,7 +996,7 @@ exports[`LocationStatusTags component renders when there are many location 1`] =
                   aria-controls="generated-id"
                   aria-current="true"
                   aria-label="Page 1 of 2"
-                  class="euiButtonEmpty euiButtonEmpty--text euiButtonEmpty--xSmall euiButtonEmpty-isDisabled euiPaginationButton euiPaginationButton-isActive euiPaginationButton--hideOnMobile"
```

Comment:
size `xs` is removed?

Reply:
Size `xs` buttons still exist, but arrow buttons in [EuiPagination now use `s`](https://github.com/elastic/eui/pull/4661/files#diff-d7acf5d494cf576fe31d241ffa50e42d5addf1a4fa18fb520e2a58aa0a9e6b51) for larger click area. There is no real change in appearance.